### DISCUSSION TITLE
Units?

### DISCUSSION THREAD

**Discussion by BillyCroan** [ianjwhite99/sync-connect-sdk#56] [Q&A] [answered]
What units are battery and fuel in?  For that matter, tire pressure as well could use units.  And odometer.  Then again latestMILEage is pretty clear, but how that that show up for a car using KM?  Does ford convert it to Mi in the cloud?

It it better to include this in the code, or documentation or both?

**Comment by BillyCroan** [comment]
I'm tempted to add these details to the docs, along with an example of the json output we get back from ford.  Also, any idea why ford makes their json with single quotes instead of doubles?  that causes problems with jq.

**Comment by HKLM** [comment, marked as answer]
This should help you. This is for a gas engine vehicle, I dont have a EV, so I have no idea of the data those vehicles show.


```json
"odometer": {
      "value": 3095.0, //<--km (Kilometers)
},
"oil": {
      "oilLife": "STATUS_GOOD",
      "oilLifeActual": 84, //<-- % (Percentage)
},
"fuel": {
      "fuelLevel": 43.695792,  //<--Liter
      "distanceToEmpty": 299.9, //<-- km (Kilometers)
},
"battery": {
	"batteryStatusActual": {
        "value": 13, //<--Volts
	}
},
"TPMS": {
      "leftFrontTirePressure": {
        "value": "441", //<--kPa (Kilopascal) 
   },
      "recommendedFrontTirePressure": {
        "value": 60 //<--PSI
   },
      "recommendedRearTirePressure": {
        "value": 65 //<--PSI
   }
},
```

Funny thing is I am in the USA and have set the preferences in the vehicle to PSI, and the only data that comes from the ford API is the recommendedTirePressure values are in PSI.  All the actual TirePressure values come in kPa units.
